### PR TITLE
Support new format to create vsphere.conf compatible with k8s 1.9

### PR DESCRIPTION
**Pull request by alesz**
Update template and required variables to create vsphere.conf compatible with Kubernetes 1.9+

### CODE REVIEW

File: roles/openshift_cloud_provider/tasks/vsphere.yml [hunk 1]
```diff
@@ -3,4 +3,4 @@
   template:
     dest: "{{ openshift.common.config_base }}/cloudprovider/vsphere.conf"
     src: vsphere.conf.j2
-  when: openshift_cloudprovider_vsphere_username is defined and openshift_cloudprovider_vsphere_password is defined and openshift_cloudprovider_vsphere_host is defined and openshift_cloudprovider_vsphere_datacenter is defined and openshift_cloudprovider_vsphere_datastore is defined
+  when: openshift_cloudprovider_vsphere_username is defined and openshift_cloudprovider_vsphere_password is defined and openshift_cloudprovider_vsphere_datacenters is defined and openshift_cloudprovider_vsphere_virtualcenters is defined
```
Comment (vrutkovs) [inline roles/openshift_cloud_provider/tasks/vsphere.yml]: Lets split this into the list:
```
when:
- openshift_cloudprovider_vsphere_username is defined
- openshift_cloudprovider_vsphere_password is defined
...
```

**Comment by vrutkovs** [comment]
bot, test pull request

**Comment by alesz** [comment]
rebased to master

**Comment by vrutkovs** [comment]
Master branch is closed! A major refactor is ongoing in devel-40. Changes for 3.x should be made directly to the latest release branch they're relevant to and backported from there.

**Pull request closed without merging**